test/lib/get-status-func-spec.js

```diff
@@ -9,8 +9,9 @@ const {
 
 const API_TYPE = 'experienceViews';
 const COMMAND_TYPE = 'views';
+const template = require('lodash-template');
 const LOCAL_STATUS_PARAMS = [ '/**/*.hbs' ];
-const REMOTE_STATUS_PARAMS = [ 'views/${viewType}s/${name}.hbs', 'body' ]; // eslint-disable-line no-template-curly-in-string
+const REMOTE_STATUS_PARAMS = [ template('views/${viewType}s/${name}.hbs'), 'body' ]; // eslint-disable-line no-template-curly-in-string
```

Comment:
technically this no longer matches what you are actually doing with experiences

Reply:
I think originally I was making these test actually test the get status function. not that it needs to be done with a resource type we have. but it did clean up the last tests.